src/libraries/System.Globalization/tests/CompareInfo/CompareInfoTests.Compare.cs

```diff
@@ -238,19 +276,26 @@ public static IEnumerable<object[]> Compare_TestData()
                 yield return new object[] { s_invariantCompare, "\u0021", "\uFF01", CompareOptions.IgnoreWidth, 0 };
                 yield return new object[] { s_invariantCompare, "\uFF66", "\u30F2", CompareOptions.IgnoreWidth, 0 };
             }
-
-            yield return new object[] { s_invariantCompare, "\uFF66", "\u30F2", CompareOptions.IgnoreSymbols, s_expectedHalfToFullFormsComparison };
-            yield return new object[] { s_invariantCompare, "\uFF66", "\u30F2", CompareOptions.IgnoreCase, s_expectedHalfToFullFormsComparison };
-            // in HybridGlobalization on Browser IgnoreNonSpace is not supported and comparison of katakana/hiragana equivalents with supportedIgnoreNonSpaceOption gives 0
-            if (!PlatformDetection.IsHybridGlobalizationOnBrowser)
-                yield return new object[] { s_invariantCompare, "\uFF66", "\u30F2", CompareOptions.IgnoreNonSpace, s_expectedHalfToFullFormsComparison };
-            yield return new object[] { s_invariantCompare, "\uFF66", "\u30F2", CompareOptions.None, s_expectedHalfToFullFormsComparison };
+            
+            if(!PlatformDetection.IsHybridGlobalizationOnOSX)
```

Comment:
Only IgnoreSymbols in the block below isn't supported right? Why are we skipping the others?

Reply:
Updated, but this case and more other cases behave differently with native api. 
I will add description in docs for these cases that will be different.